modules/nw-ingress-setting-internal-lb.adoc

```diff
@@ -0,0 +1,30 @@
+[id="nw-ingress-setting-internal-lb_{context}"]
+= Configuring ingress controller to use an internal load balancer
+
+When creating an IngressController on cloud platforms, the IngressController is
+published by a public cloud load balancer by default.
+
+Users can now publish IngressControllers with internal cloud load balancers. For example:
+
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: internal
+spec:
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: Internal
+----
+
+See the link:https://kubernetes.io/docs/concepts/services-networking/#internal-load-balancer.[Kubernetes Services documentation]
```

Comment:
The link has a spurious "." in it.  Also, the documentation has apparently moved and is now at the following URL: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer